schema/v2/card_schema.json

```diff
@@ -0,0 +1,328 @@
+{
+  "$id": "https://github.com/netrunnerdb/netrunner-cards-json/blob/master/schema/v2/card_schema.json",
+  "$schema": "https://json-schema.org/draft/2020-12/schema",
+  "type": "object",
+  "additionalProperties": false,
+  "allOf": [
+    {
+      "title": "Faction id must be valid for the side.",
+      "type": "object",
+      "oneOf": [
+        {
+          "properties": {
+            "side_id": {
+              "const": "corp"
+            },
+            "faction_id": {
+              "enum": [
+                "haas-bioroid",
+                "jinteki",
+                "nbn",
+                "neutral-corp",
+                "weyland-consortium"
```

Comment:
Should these use underscores instead of hyphens?

Reply:
leaving broader formatting changes to their own PR.